docs/server/source/production-deployment-template/workflow.rst

```diff
@@ -53,6 +53,26 @@ Similarly, other instances must also have unique names in the cluster.
 #. Name of the MongoDB backup agent instance (``mdb-bak-instance-*``)
 
 
+☐ Generate four keys and corresponding certificate signing requests (CSRs):
+
+#. Server Certificate (a.k.a. Member Certificate) for the MongoDB instance
+#. Client Certificate for BigchainDB Server to identify itself to MongoDB
+#. Client Certificate for MongoDB Monitoring Agent to identify itself to MongoDB
+#. Client Certificate for MongoDB Backup Agent to identify itself to MongoDB
+
```

Comment:
I would not use the word `identify` as certificates help in identification as well as secure communications.
We had secure communications before enabling auth on MongoDB.